ui/DesignSystem/Component/Notification.svelte

```diff
@@ -65,9 +70,15 @@
   .info .action-divider {
     background-color: var(--color-foreground-level-6);
   }
+
+  .variant-primary .action-divider {
+    background-color: var(--color-primary-level-1);
+  }
```

Comment:
Is this the desired color? It looks brighter than for the other notifications:
<img width="444" alt="Screenshot 2021-01-21 at 13 05 30" src="https://user-images.githubusercontent.com/16262137/105348976-58c8e300-5be9-11eb-8320-a51a0194ea21.png">

Reply:
I0ve checked. This is the correct color, taken directly from the figma design.

Reply:
@brandonhaslegs Maybe you can take a look if this needs work (in a different PR). In the Dark-theme, the vertical dividers also are not all the same.